content/en/docs/concepts/architecture/garbage-collection.md

```diff
@@ -137,6 +137,13 @@ collection, which deletes images in order based on the last time they were used,
 starting with the oldest first. The kubelet deletes images
 until disk usage reaches the `LowThresholdPercent` value.
 
+As of v1.29.0, the kubelet supports a new configuration field:
+
+* `ImageMaximumGCAge`
+
+This field, when enabled with the alpha feature gate of the same name, will set the
+maximum time an image can be unused for, regardless of disk usage.
+
```

Comment:
Please move this into a new section that comes right at the end of the `## Configuring garbage collection` section. 

I suggest, based on our style guide and conventions:
```diff
-As of v1.29.0, the kubelet supports a new configuration field:
-
-* `ImageMaximumGCAge`
-
-This field, when enabled with the alpha feature gate of the same name, will set the
-maximum time an image can be unused for, regardless of disk usage.
+{{< feature-state for_k8s_version="v1.29" state="alpha" >}}
+
+As an alpha feature, you can specify the maximum time a local image can be unused for,
+regardless of disk usage. This is a kubelet setting that you configure for each node.
+To configure the setting, enable the `ImageMaximumGCAge`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) for the kubelet,
+and also set a value for the `ImageMaximumGCAge` field in the kubelet configuration file.
+The value is specified as a Kubernetes _duration_; for example, you can set the configuration
+field to `3d12h`, which means 3 days and 12 hours.
```